src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2016, Ford Motor Company
```

Comment:
@pold500 Minor note: when you add new file, please check the copyright date and update it if necessary. (Please check in all your other files)
"Copyright (c) 2017, Ford Motor Company"